### PR TITLE
[MRG] [TST] MNT remove unused variables

### LINKED ISSUE #5571: Unused variables in test_brainvision.py
https://github.com/mne-tools/mne-python/blob/fd23ba8bfc1f11dd046abcfa89868f7dd8569d76/mne/io/brainvision/tests/test_brainvision.py#L37

https://github.com/mne-tools/mne-python/blob/fd23ba8bfc1f11dd046abcfa89868f7dd8569d76/mne/io/brainvision/tests/test_brainvision.py#L41

https://github.com/mne-tools/mne-python/blob/fd23ba8bfc1f11dd046abcfa89868f7dd8569d76/mne/io/brainvision/tests/test_brainvision.py#L114

### PR DESCRIPTION
**Pull request by massich**
fixes #5571 

To @palday, @teonbrooks do you need the `xxxxx.vmrk`?
To @agramfort, I guess that testing `channels` was enough.

### CODE REVIEW

**Comment by teonbrooks** [comment]
i'm not familar with ``test_old_layout_latin1_software_filter.vmrk``. according to the `git blame` @palday added it. maybe he could share what this file is supposed to do

**Comment by massich** [comment]
@teonbrooks you added `vmrk_v2_path` by pushing 481f250df4d7e6454c3a923cf81b28b02b5c522b to master

**Comment by massich** [comment]
@palday yes `vmrk_old_path` was added in #3532, but was unused already there.

**Comment by palday** [comment]
Hence my comment on it being a copy-and-paste job from existing tests -- it looks like other tests needed the path for the vhdr, vmrk and data files and so specified all three. For the tests in question, the vmrk path wasn't necessary, but symmetry from the copy-paste kept it in.

File: mne/io/brainvision/tests/test_brainvision.py [hunk 1]
```diff
@@ -111,7 +108,6 @@ def test_vhdr_codepage_ansi():
                     line = b'Codepage=ANSI\n'
                 fout.write(line)
     raw = read_raw_brainvision(ansi_vhdr_path, event_id=event_id)
-    data_new, times_new = raw[:]
```
Comment (agramfort) [inline mne/io/brainvision/tests/test_brainvision.py]: I would actually keep it and test something on data_new and times_new as I think this reading part was broken before the test was added

**Comment by agramfort** [comment]
thx @massich

**Pull request merged**